proxy/outprocess/inc/module_loaders/outprocess_loader.h

```diff
@@ -45,10 +45,12 @@ typedef struct OUTPROCESS_LOADER_ENTRYPOINT_TAG
     STRING_HANDLE control_id;
     /** @brief The URI for the gateway message channel.*/
     STRING_HANDLE message_id;
+	/** @brief controls timeout for ipc retries. */
+	int default_wait;
 } OUTPROCESS_LOADER_ENTRYPOINT;
 
 /** @brief      The API for the out of process proxy module loader. */
-MOCKABLE_FUNCTION(, GATEWAY_EXPORT const MODULE_LOADER*, OutprocessLoader_Get);
+MOCKABLE_FUNCTION(, extern const MODULE_LOADER*, OutprocessLoader_Get);
```

Comment:
It solved a problem I shouldn't have been having, and I accidentally checked it in.  Fixed